backend/webapi/Features/AccessRequests/HcimAccountTransfer.cs

```diff
@@ -75,14 +75,19 @@ public class CommandHandler : ICommandHandler<Command, IDomainResult<Model>>
 
         public async Task<IDomainResult<Model>> HandleAsync(Command command)
         {
+            var userIds = await this.context.Credentials
+                    .Where(credential => credential.PartyId == command.PartyId)
+                    .Select(credential => credential.UserId)
+                    .ToListAsync();
+
             var dto = await this.context.Parties
                 .Where(party => party.Id == command.PartyId)
                 .Select(party => new
                 {
                     AlreadyEnroled = party.AccessRequests.Any(request => request.AccessTypeCode == AccessTypeCode.HcimAccountTransfer
                         || request.AccessTypeCode == AccessTypeCode.HcimEnrolment),
                     DemographicsComplete = party.Email != null && party.Phone != null,
-                    party.UserId,
+                    userIds,
```

Comment:
this can just be `UserIds = party.Credentials.Select(credential => credential.UserId)`